install/kubernetes/helm/istio/charts/prometheus/templates/configmap.yaml

```diff
@@ -251,19 +251,62 @@ data:
         action: replace
         target_label: kubernetes_name
 
-    # Example scrape config for pods
-    - job_name: 'kubernetes-pods'
+    - job_name: 'kubernetes-pods-non-istio'
       kubernetes_sd_configs:
```

Comment:
@mandarjog - can you find someone to review this configmap?  Looks like greek to me ;)